kroxylicious-kms/src/main/java/io/kroxylicious/kms/service/AwsNotImplementException.java

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
This file is in the wrong package.  It is a concern of AWS only.

Reply:
Moved to `io.kroxylicious.kms.provider.aws.kms`